vscode-ext/src/CaesarClient.ts

```diff
@@ -113,6 +113,17 @@ export class CaesarClient {
                 this.needsRestart = true;
             }
         }));
+
+        // listen to onDidSaveTextDocument events
+        const autoVerify: string = CaesarConfig.get(ConfigurationConstants.automaticVerification);
```

Comment:
This will only check the automatic verification setting once when the extension is loaded, as opposed to every time the server is reloaded (which happens when the config changes). I have two options: a) just save the `Disposable` from `onDidSaveTextDocument` and dispose of the callback when the server is stopped, or b) do the check inside the callback (but then check for languageId first, only then do the config check).